docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -273,6 +273,75 @@ public class SubjectExposingResource {
 
 CAUTION: Please refer to the xref:security-built-in-authentication-support-concept.adoc#proactive-authentication[Proactive Authentication] section of the Built-In Authentication Support guide if you plan to use standard security annotations on IO thread.
 
+The `@RolesAllowed` annotation value supports <<config-reference#property-expressions,Property Expressions>> including default values and nested Property Expressions.
+Configuration properties used with the annotation are resolved at runtime and supports configuration profiles.
```

Comment:
```suggestion
Configuration properties used with the annotation are resolved at runtime.
```

Reply:
I thought the note about configuration profile does not clarify the message that these properties are resolved at runtime - the profiles are part of the Config system

Reply:
Looking at it now, I think examples below with 2 configuration profiles are enough. I removed the note, thank you.